data/asd/asd.v3.mf.xml

```diff
@@ -45,8 +45,8 @@
                         <direction>1010101010101010</direction>
                     </GeographicalPath>
                 </regions>
-                <sspMsgRights1>2</sspMsgRights1>
-                <sspMsgRights2>3</sspMsgRights2>
```

Comment:
It would be helpful if you could update the README.md in this data folder to note how/where all files in asd folder generated/used?

Reply:
Documentation has been added for which data files are used and where. These don't appear to be referenced anywhere.